ci/jenkins/test-mc.sh

```diff
@@ -128,12 +133,10 @@ function cleanup_multicluster_antrea {
     echo "====== Cleanup Antrea controller and agent ======"
     kubeconfig=$1
     kubectl get pod -n kube-system -l component=antrea-agent --no-headers=true $kubeconfig | awk '{print $1}' | while read AGENTNAME; do
-       kubectl exec $AGENTNAME -c antrea-agent -n kube-system ${kubeconfig} -- ovs-vsctl del-port br-int gw0 || true
+       kubectl exec $AGENTNAME -c antrea-agent -n kube-system ${kubeconfig} -- ovs-vsctl del-port br-int antrea-gw0 || true
```

Comment:
Actually this line is meant to remove "gw0" exactly because the gateway interface was renamed from "gw0" to "antrea-gw0", but around the time when the change is made, some PRs created antrea-gw0 while other created gw0, causing some networking issues so the line was added to cleanup the old port regardless. It has been 1.5 years so I think it's ok to remove the step now.

Reply:
Ok, removed.